cloudsync/tests/fixtures/util.py

```diff
@@ -65,7 +65,10 @@ def is_found(files: Sequence[WaitForArg], providers: Tuple[Provider, Provider],
 
 
 class RunUntilHelper:
+    default_timeout = 10  # seconds
+
     def run_until_clean(self: Any, timeout=TIMEOUT):
+        timeout = 1000;
```

Comment:
guessing this should be cleaned up?

Reply:
Yeah, I always put a `;` at the ends of lines that I want to remove before the code review, because lint is supposed to complain about it, but I guess lint isn't configured to complain about semicolons in this repo?